xml/ssh_banners.xml

```diff
@@ -778,10 +778,11 @@ fingerprint SSH servers.
       <param pos="0" name="os.product" value="Windows"/>
    </fingerprint>
    
-   <fingerprint pattern="^Comware-(\d+\.?\d*\.?\d*)$">
+   <fingerprint pattern="^(?:SSH-[1|2]\.\d{1,2}-)?Comware-(\d+\.?\d*\.?\d*)$">
```

Comment:
I think I see what I did. I pulled some banner examples from the console which displays the full banner and others from scan logs which display just display the match. I'll test it and resubmit.